test/unit/yesno.spec.ts

```diff
@@ -615,5 +609,45 @@ describe('Yesno', () => {
         expect(yesno.intercepted()).to.have.lengthOf(3);
       });
     });
+
+    describe('#ignore', () => {
+      it('allows ignoring a mocked url', async () => {
+        yesno.mock([createMock(), createMock(), createMock()]);
```

Comment:
I'd like to move away from anywhere we see this ` yesno.mock([createMock(), createMock(), createMock()]);`. If we create three mocks let's always make them unique in tests so our assertions can be linked back to the mock defined.

In my opinion having three mocks that can't be differentiated in the assertions makes these tests weaker.